<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><title>Function reference â€¢ TS</title><script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet"><script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- bootstrap-toc --><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@v1.0.1/dist/bootstrap-toc.min.js" integrity="sha256-4veVQbu7//Lk5TSmc7YV48MxtMy98e26cf5MrgZYnwo=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Function reference"><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-dark navbar-expand-lg bg-primary"><div class="container">
    
    <a class="navbar-brand me-2" href="../index.html">TS</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">1.0</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto"><li class="active nav-item">
  <a class="nav-link" href="../reference/index.html">Reference</a>
</li>
      </ul><form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../search.json" id="search-input" placeholder="Search for" autocomplete="off"></form>

      <ul class="navbar-nav"></ul></div>

    
  </div>
</nav><div class="container template-reference-index">
<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="" class="logo" alt=""><h1>Function reference</h1>
    </div>

    <div class="section level2">
      <h2 id="all-functions">All functions<a class="anchor" aria-label="anchor" href="#all-functions"></a></h2>
      
      

      
    </div><div id="" class="section level2">
      
      
      

      <dl><dt>
          
          <code><a href="TS_DEenrichRPlot.html">TS_DEenrichRPlot()</a></code> 
        </dt>
        <dd>A wrapper around the DEnrichRPlot from the 'Seurat' package</dd>
      </dl><dl><dt>
          
          <code><a href="TS_FindConservedMarkers.html">TS_FindConservedMarkers()</a></code> 
        </dt>
        <dd>Find conserved markers within two groups</dd>
      </dl><dl><dt>
          
          <code><a href="TS_Find_DEGs_and_overlap.html">TS_Find_DEGs_and_overlap()</a></code> 
        </dt>
        <dd>Find the upregulated DEGs for each cluster, comparing two clusters. As well as the conserved markers between them.</dd>
      </dl><dl><dt>
          
          <code><a href="TS_IGHV_usage_prop_with_stats.html">TS_IGHV_usage_prop_with_stats()</a></code> 
        </dt>
        <dd>Plots IGHV usage between groups together with Fishers Exact Test</dd>
      </dl><dl><dt>
          
          <code><a href="TS_Seurat_from_file.html">TS_Seurat_from_file()</a></code> 
        </dt>
        <dd>Process CellRanger output files with Seurat</dd>
      </dl><dl><dt>
          
          <code><a href="TS_calculate_ratios.html">TS_calculate_ratios()</a></code> 
        </dt>
        <dd>Calculate light chain ratios of a Seurat object, requires light chain data in metadata "light.chain"</dd>
      </dl><dl><dt>
          
          <code><a href="TS_cellnumbers_per_PC_and_resolution.html">TS_cellnumbers_per_PC_and_resolution()</a></code> 
        </dt>
        <dd>Calculate cellnumbers per cluster for each combination of 'number or PCs' and 'resolution'</dd>
      </dl><dl><dt>
          
          <code><a href="TS_compare_PC_and_resolution.html">TS_compare_PC_and_resolution()</a></code> 
        </dt>
        <dd>Compare number of clusters for combinations of number of PCs and resolution</dd>
      </dl><dl><dt>
          
          <code><a href="TS_count_VDJ.html">TS_count_VDJ()</a></code> 
        </dt>
        <dd>Not sure if this function is desirable anymore. So use it at your own risk</dd>
      </dl><dl><dt>
          
          <code><a href="TS_count_VDJ_7.0.html">TS_count_VDJ_7.0()</a></code> 
        </dt>
        <dd>Count VDJ from Cellranger 7.0</dd>
      </dl><dl><dt>
          
          <code><a href="TS_dataframe_to_fasta.html">TS_dataframe_to_fasta()</a></code> 
        </dt>
        <dd>Converts a dataframe and saves it in Fasta format</dd>
      </dl><dl><dt>
          
          <code><a href="TS_export_cluster_ids.html">TS_export_cluster_ids()</a></code> 
        </dt>
        <dd>Export cell barcodes for each cluster</dd>
      </dl><dl><dt>
          
          <code><a href="TS_export_named_ids.html">TS_export_named_ids()</a></code> 
        </dt>
        <dd>Export identities for each cluster, which are not numerical</dd>
      </dl><dl><dt>
          
          <code><a href="TS_extract_ig_mean_median_rank_per_PC.html">TS_extract_ig_mean_median_rank_per_PC()</a></code> 
        </dt>
        <dd>Extract the mean and median values of Ig-genes from a list of PCA feature loadings</dd>
      </dl><dl><dt>
          
          <code><a href="TS_extract_ig_ranks_per_PC.html">TS_extract_ig_ranks_per_PC()</a></code> 
        </dt>
        <dd>Extract the 'rank' for each of the Ig-genes</dd>
      </dl><dl><dt>
          
          <code><a href="TS_extract_ranked_genes_per_PC.html">TS_extract_ranked_genes_per_PC()</a></code> 
        </dt>
        <dd>Extract an ordered list of feature importance for each of the first 50 principal components</dd>
      </dl><dl><dt>
          
          <code><a href="TS_featureplot_to_pdf.html">TS_featureplot_to_pdf()</a></code> 
        </dt>
        <dd>Export featureplot as pdf to your working directory</dd>
      </dl><dl><dt>
          
          <code><a href="TS_featureplot_to_pdf_A4.html">TS_featureplot_to_pdf_A4()</a></code> 
        </dt>
        <dd>Export featureplot to pdf in A4 format</dd>
      </dl><dl><dt>
          
          <code><a href="TS_featureplot_to_pdf_no_leg_no_axes.html">TS_featureplot_to_pdf_no_leg_no_axes()</a></code> 
        </dt>
        <dd>Export featureplot to pdf without legend and axes</dd>
      </dl><dl><dt>
          
          <code><a href="TS_featureplots.html">TS_featureplots()</a></code> 
        </dt>
        <dd>Output one or multiple featureplots</dd>
      </dl><dl><dt>
          
          <code><a href="TS_filter_marker_list.html">TS_filter_marker_list()</a></code> 
        </dt>
        <dd>Filter list of markers from FindMarkers function, needs to be output into new df</dd>
      </dl><dl><dt>
          
          <code><a href="TS_plot_3D.html">TS_plot_3D()</a></code> 
        </dt>
        <dd>Plot your Seurat object in 3D.</dd>
      </dl><dl><dt>
          
          <code><a href="TS_plot_counts_boxplot.html">TS_plot_counts_boxplot()</a></code> 
        </dt>
        <dd>Plot counts for a gene in a Seurat object as boxplot</dd>
      </dl><dl><dt>
          
          <code><a href="TS_plot_counts_violin.html">TS_plot_counts_violin()</a></code> 
        </dt>
        <dd>Plot counts for a gene in a Seurat object as violinplot</dd>
      </dl><dl><dt>
          
          <code><a href="TS_plot_enrichR.html">TS_plot_enrichR()</a></code> 
        </dt>
        <dd>Plot the output from TS_run_enrichR</dd>
      </dl><dl><dt>
          
          <code><a href="TS_project_scRNAseq_dataset.html">TS_project_scRNAseq_dataset()</a></code> 
        </dt>
        <dd>Projects one Seurat object onto another</dd>
      </dl><dl><dt>
          
          <code><a href="TS_project_scRNAseq_dataset_export_query.html">TS_project_scRNAseq_dataset_export_query()</a></code> 
        </dt>
        <dd>Projects one Seurat object onto another and exports resulting dataframe</dd>
      </dl><dl><dt>
          
          <code><a href="TS_run_enrichR.html">TS_run_enrichR()</a></code> 
        </dt>
        <dd>Wrapper around the enrichR package.</dd>
      </dl><dl><dt>
          
          <code><a href="TS_seurat_object_stats.html">TS_seurat_object_stats()</a></code> 
        </dt>
        <dd>Prints stats about your Seurat object</dd>
      </dl><dl><dt>
          
          <code><a href="TS_svg_from_featureplot.html">TS_svg_from_featureplot()</a></code> 
        </dt>
        <dd>Export featureplot as svg to your working directory</dd>
      </dl><dl><dt>
          
          <code><a href="TS_svg_from_plot.html">TS_svg_from_plot()</a></code> 
        </dt>
        <dd>Export any plot as svg</dd>
      </dl><dl><dt>
          
          <code><a href="TS_visdimloadings.html">TS_visdimloadings()</a></code> 
        </dt>
        <dd>Plot gene contribution to each principal component</dd>
      </dl></div>
  </main></div>


    <footer><div class="pkgdown-footer-left">
  <p></p><p>Developed by Timothy Sundell.</p>
</div>

<div class="pkgdown-footer-right">
  <p></p><p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
</div>

    </footer></div>

  

  

  </body></html>

